spec/plans/report.fmf

```diff
@@ -128,6 +128,12 @@ description:
         # Optionally set environment variables according to TMT_PLUGIN_REPORT_REPORTPORTAL_${OPTION}
         export TMT_PLUGIN_REPORT_REPORTPORTAL_URL=${url-to-RP-instance}
         export TMT_PLUGIN_REPORT_REPORTPORTAL_TOKEN=${token-from-RP-profile}
+
+        # Bolean options are activated with value of 1:
+        TMT_PLUGIN_REPORT_REPORTPORTAL_SUITE_PER_PLAN=1
+
+      - NOTE: All options are supported in the format demonstrated above to enable execution purely via metadata and/or environment variables (e.g. Testing Farm)
```

Comment:
This form is probably going to cause you trouble. First, I think it's behind the failed readthedocs build because written like this, the YAML parser ingests it as `[..., {'NOTE': 'All options...'}, ...]` which is not correct. Second, when rendered into docs, the note will look like yet another example, with much less connection to other examples. It would be better to include this note in the `description` block.